packages/core/addon/manifests/bar-chart.js

```diff
@@ -32,6 +32,9 @@ export default ManifestBase.extend({
    * @return {Boolean} - visualization type is valid
    */
   typeIsValid(request) {
-    return this.hasMetric(request) && (this.hasGroupBy(request) || this.hasMultipleTimeBuckets(request));
+    return (
+      this.hasMultipleMetrics(request) ||
```

Comment:
what request do we not want a bar chart?

Reply:
no metrics obviously, or no grouping (one time bucket and no dimensions) otherwise it's a single column (?)
I usually tend to whitelist things rather than blacklist to not have surprises